CONTRIBUTING.md

```diff
@@ -169,6 +171,10 @@ When declaring a record in XML,
 </record>
 ```
 
+### Records
+
+* For records of model `ir.filters` use explicit `user_id` field.
```

Comment:
I mean, this phrase does not make obvious that we have to write `<field name="user_id" eval="False"/>`, rather I first understood that we had to actually put an existing `user_id`.